_data/packages.yml

```diff
@@ -425,9 +465,6 @@
       github_username: vn-ki
   repository_link: https://github.com/sunpy/sunpy
   version_submitted: 5.0.1
-  sunpy: true
```

Comment:
I didn't touch this in #520 but does seem weird it is removed here regardless.

Reply:
i may have added this manually back when i was playing with what partner community cards would look like. i wouldn't put it past me to do that and we probably could use blame to confirm. but i suspect I added it. Then I realized sunpy is associated with pyhc in some ways so we may want to have multiple affiliations for a single package in some cases.